frontend/src/features/map/index.tsx

```diff
@@ -253,6 +258,10 @@ export function Map({ isSuperUser }) {
       {isRecentActivityEnabled ? <RecentActivityLayerEvents /> : null}
       {/* @ts-ignore */}
       {isRecentActivityEnabled ? <RecentActvityOverlay /> : null}
+      {/* @ts-ignore */}
+      {isRecentActivityEnabled && displayRecentActivityLayer && !isRecentActivityDialogVisible ? (
```

Comment:
suggestion : Pas besoin de mettre des ternaire : 
```suggestion
      {isRecentActivityEnabled && displayRecentActivityLayer && !isRecentActivityDialogVisible && (<RecentActivityLegend forceVisibility />)
```

Reply:
On est obligé de passer une ternaire, sinon on a cette erreur : 
```
Type 'false | Element' is not assignable to type 'ReactElement<BaseMapChildrenProps, string | JSXElementConstructor<any>> | null'.
  Type 'boolean' is not assignable to type 'ReactElement<BaseMapChildrenProps, string | JSXElementConstructor<any>>'
  ```